### PR TITLE
[12.0] Migrate accruals modules to v12

### PR DESCRIPTION
**Pull request by alexis-via**
Also up-port improvements in account_cutoff_base and account_cutoff_prepaid from v10 to v12, and add entries in accounting config page.

### CODE REVIEW

File: account_cutoff_accrual_base/models/account_cutoff.py [hunk 1]
```diff
@@ -13,28 +12,14 @@ class AccountCutOff(models.Model):
...
-        cutoff_type = self.env.context.get('default_type')
-        company = self.env.user.company_id
-        default_journal = company.default_cutoff_journal_id
-        if cutoff_type == 'accrued_expense':
-            journal = company.default_accrual_expense_journal_id or\
-                default_journal
-        elif cutoff_type == 'accrued_revenue':
-            journal = company.default_accrual_revenue_journal_id or\
-                default_journal
-        return journal
```
Comment (sbidoul) [inline account_cutoff_accrual_base/models/account_cutoff.py]: Hi @alexis-via this method and associated default values are used in particular in `account_invoice_accrual`.
Can we preserve them? 

cc/ @sbejaoui

Comment (sbejaoui) [reply account_cutoff_accrual_base/models/account_cutoff.py]: Hi @alexis-via,
I created a [PR](https://github.com/akretion/account-closing/pull/2) to add missing fields.
Can you check it please.

Comment (alexis-via) [reply account_cutoff_accrual_base/models/account_cutoff.py]: I wanted to simplify all these additionnal settings and keep the ones that are really used by a majority of users... but your collegues convinced me :-(

**Comment by sbidoul** [comment]
@alexis-via could you attend the tiny lint error?

**Pull request merged**